subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/maven/MixedMavenAndIvyModulesIntegrationTest.groovy

```diff
@@ -206,14 +206,16 @@ dependencies {
         }
     }
 
-    def "selects correct configuration of ivy module when dependency from consuming maven module is substituted"() {
-        def notRequired = ivyRepo.module("org.test", "ignore-me", "1.0")
+    def "selects default configuration of ivy module when dependency from consuming maven module is substituted"() {
```

Comment:
:o: I'd be happy to have a bit more details about this change, since I don't immediately understand it

Reply:
Effectively this test was _verifying_ the bug. That is it asserted that using a force would still trigger the legacy behavior. That makes no sense. So instead we now verify that with a force (or any substitution) we still have the behavior as described in other tests where a Maven dependency pulls in an Ivy dependency.

Reply:
I see, thanks for the explanation.